files/passwd

```diff
@@ -0,0 +1 @@
+carosadm:x:200:37:CarOS Operator:/home/carosadm:
```

Comment:
"CarOS Administrator" in recipes-caros/carosadm/carosadm_1.0.0.bb which is more fitting.